_core/smart/neon-node.md

```diff
@@ -31,22 +31,10 @@ git clone https://github.com/blockstack/stacks-blockchain.git
 cd stacks-blockchain
 ```
 
-### Configure your node to connect to the Neon network
+Install the Stacks node by running:
 
-You'll need to update your node's configuration to connect to the public Neon testnet.
-
-In the `stacks-blockchain` repository, open up the file `testnet/follower-conf.toml`.
-
-First, find the `bootstrap_node` line in the `[node]` section, and update it to the following:
-
-```toml
-bootstrap_node = "048dd4f26101715853533dee005f0915375854fd5be73405f679c1917a5d4d16aaaf3c4c0d7a9c132a36b8c5fe1287f07dad8c910174d789eb24bdfb5ae26f5f27@35.245.47.179:20444"
-```
-
-Next, find the uncommented `peer_host` line in the `[burnchain]` section, and update it to:
-
-```toml
-peer_host = "35.245.47.179"
+```bash
+cargo install --path ./testnet
```

Comment:
nit: running this takes minutes for me, it's compiling here as well (166 modules).